lib/ansible/modules/crypto/openssl_certificate_info.py

```diff
@@ -238,10 +238,45 @@
                  or not.
     returned: success
     type: dict
+subject_key_identifier:
+    description:
+        - The CSR's subject key identifier.
+        - The identifier is returned in hexadecial, with C(:) used to separate bytes.
+        - Is C(none) if the C(SubjectKeyIdentifier) extension is not present.
+    returned: success and if the pyOpenSSL backend is I(not) used
+    type: str
+    sample: '00:11:22:33:44:55:66:77:88:99:aa:bb:cc:dd:ee:ff:00:11:22:33'
+    version_added: "2.9"
+authority_key_identifier:
+    description:
+        - The CSR's authority key identifier.
+        - The identifier is returned in hexadecial, with C(:) used to separate bytes.
+        - Is C(none) if the C(AuthorityKeyIdentifier) extension is not present.
+    returned: success and if the pyOpenSSL backend is I(not) used
+    type: str
+    sample: '00:11:22:33:44:55:66:77:88:99:aa:bb:cc:dd:ee:ff:00:11:22:33'
+    version_added: "2.9"
+authority_cert_issuer:
+    description:
+        - The CSR's authority cert issuer as a list of strings.
+        - Is C(none) if the C(AuthorityKeyIdentifier) extension is not present.
+    returned: success and if the pyOpenSSL backend is I(not) used
+    type: list
+    sample: "[DNS:www.ansible.com, IP:1.2.3.4]"
+    version_added: "2.9"
+authority_cert_serial_number:
```

Comment:
As a general comment, I think it would maybe make more sense to have naming conventions:
issuer_subject_key_identifier
issuer_subject_name
issuer_serial_number

As that's typically how that information is presented about a certificate. I haven't finished reviewing the changes though, so I realize there may be backwards compatibility/consistency issues leading to the authority_ naming convention.

Reply:
After reviewing the rest I realize this was for consistency with the CSR additions.

Reply:
I mainly stuck to the names from the RFC (https://tools.ietf.org/html/rfc5280#section-4.2.1.1). There's no backwards compatibility to be considered here, so we can also change the names, but I'm not sure whether it's a good idea to choose something completely different.

Reply:
That's what I get for writing comments on the doc before actually finishing reading all the code - I forgot those pieces of information are even defined, so I was thinking that it was identifying issuer based on key identifier, then displaying information about the issuer from _there_. Thumbs up on the naming.

Reply:
Interestingly, the cryptography.io docs were very confusing on `authority_cert_issuer`. The type was specified as `GeneralNames`, but the description says it is a `Name` ([source](https://cryptography.io/en/latest/x509/reference/#cryptography.x509.AuthorityKeyIdentifier.authority_cert_issuer)). I skimmed over the type and started implementing the `Name` approach, only then to find out during testing that it didn't want that. The result is pyca/cryptography#4973 :)